### PR TITLE
chore: license headers

### PR DESCRIPTION
**Pull request by amrbashir**

### CODE REVIEW

File: LICENSE.spdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+PackageLicenseDeclared: MIT
+PackageCopyrightText: 2022-2022, The Tauri Programme in the Commons Conservancy
+PackageSummary: <text>Tauri is a rust project that enables developers to make secure
+and small desktop applications using a web frontend.
+                </text>
+PackageComment: <text>The package includes the following libraries; see
+Relationship information.
+                </text>
+Created: 2019-05-20T09:00:00Z
+PackageDownloadLocation: git://github.com/tauri-apps/tauri
```
Comment (lucasfernog) [inline LICENSE.spdx]: do we need to change this URL @nothingismagick ?

Comment (amrbashir) [reply LICENSE.spdx]: I forgot to change this whole file actually, the package name, summary, download location are all wrong

Comment (lucasfernog) [reply LICENSE.spdx]: I think we have the same problem on **all** repos :( we've been just copy pasting it

Comment (lucasfernog) [reply LICENSE.spdx]: and there's also some places where it isn't being added to the npm/crates.io package

Comment (amrbashir) [reply LICENSE.spdx]: yeah I fixed a couple of these repos yesterday but as I said someone with licensing knowledge should go through all repos.

Comment (amrbashir) [reply LICENSE.spdx]: > and there's also some places where it isn't being added to the npm/crates.io package

That I didn't fixt

Comment (amrbashir) [reply LICENSE.spdx]: Just checked tao, wry, win7-notifications, create-tauri-app, window-vibrancy and window-shadow and they all include this file in the publish packages

**Review by nothingismagick — APPROVED** [review]

**Pull request merged**